extensions/wikia/CategoryExhibition/CategoryExhibitionSection.class.php

```diff
@@ -46,7 +46,12 @@ public function isCategoryExhibitionEnabled() {
 			$this->categoryExhibitionEnabled = false;
 			$oTmpArticle = new Article( $this->categoryTitle );
 			if ( !is_null( $oTmpArticle ) ) {
-				$rdTitle = $oTmpArticle->getRedirectTarget();
+				if ( $this->categoryTitle->isRedirect() ) {
+					$rdTitle = $oTmpArticle->getRedirectTarget();
```

Comment:
However, it is a decision that was made to keep the behavior of this check consistent with how CategoryExhibition works